test/ctia/entity/feed_test.clj

```diff
@@ -252,13 +171,16 @@
                          :feed_view_url)))
 
           (let [feed-view-url (:feed_view_url updated-feed)
-                response (client/get feed-view-url
-                                     {:as :json})
+                response  (with-redefs [sut/fetch-limit 19] ;; < |judgements| and not a multipe of nb shards (5).
```

Comment:
if we revert back the default sort by to `_doc` this test fails because for each `_doc` value, we have 5 (number of shards) instances that share this value. Thus when sorting on `_doc` with a page size that is not a multiple of 5, we will not return the 5 documents for the last page value and then skip the missing documents with `search_after` at next page call.

Reply:
It is possible (although not a particularly common technique) to `with-redefs [default-sort-field :_doc]`, and run some "negative" tests, to prove that the tests fail with the old value.

That way, we gain confidence:

* that the tests are passing for the precise reason we believe they are
  * Maybe something else is performing the desired fix
* that the test is valid
  * maybe it would pass in all cases

A handy trick:

```clj
(is (cond-> (= (count judgements)
               (count (:judgements response-body)))
      negative? not))
```